mylasqlanon.py

```diff
@@ -109,20 +109,20 @@
                 logger.debug (f"No change indicated for {row} {col_name}")
                 continue
             # Faker has no parameters
-            if "faker" in mod_name:
+            if "faker" == mod_name:
                 logger.debug(f"Transforming {col_name} with Faker")
                 if method_name == "date_time_on_date":
                     df.at[row, col_name] = dyn_func(df.at[row, col_name])
                 else:
                     df.at[row, col_name] = dyn_func()
-            elif "ffx" in mod_name:
+            elif "ffx" == mod_name:
                 try:
                     logger.debug(f"Transforming {col_name} with FFX")
                     df.at[row, col_name] = dyn_func(df.at[row, col_name], addition=ID_ADDITION)
                 except ValueError:
                     logger.exception(f"Problem converting {df.at[row, col_name]}")
                     logger.info(np.isnan(df.at[row, col_name]))
-            elif "TODO" in "mod_name":
+            elif "TODO" == "mod_name":
```

Comment:
It should just be`mod_name` here, right?

Reply:
Yeah I don't actually have TODO anymore, I think we can just remove this.